kubernetes/helm/startree-thirdeye/templates/quickstart/job.yaml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2022 StarTree Inc
+#
+# Licensed under the StarTree Community License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at http://www.startree.ai/legal/startree-community-license
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+# either express or implied.
+# See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+
+{{- if .Values.quickstart.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: quickstart-datasets
+spec:
+  template:
+    metadata:
+      name: quickstart-datasets
+    spec:
+      initContainers:
+      - name: wait-for-pinot
+        image: alpine/curl:latest
+        env:
+          - name: CONTROLLER_PROTOCOL
+            value: {{ (.Values.quickstart.pinot).controllerProtocol | default "http" | quote }}
+          - name: CONTROLLER_HOST
+            value: {{ (.Values.quickstart.pinot).controllerHost | default "pinot-pinot-controller-headless" | quote }}
+          - name: CONTROLLER_PORT
+            value: {{ (.Values.quickstart.pinot).controllerPort | default "9000" | quote }}
+        command: ['sh', '-c', 'until curl -f $CONTROLLER_PROTOCOL://$CONTROLLER_HOST:$CONTROLLER_PORT/health; do echo waiting for pinot; sleep 2; done']
```

Comment:
I think a timeout is necessary. 
At first my configuration was incorrect and the pod got blocked in init state forever, without much feedback
I think a 5 minutes timeout then an error that prints the pinot configuration and that says the pinot instance could not be reached would help.

Reply:
Valid point. Was worried about the job exceeding the retry limits before the issue being resolved on pinot, then we need to manually trigger it. But if the configs that are passed are in itself incorrect then anyways we need to manually delete the old job first before pushing the upgrade as Job resource is read only in k8s. But we should be able to hack our way through both the cases by setting `quickstart.enabled=false` and then set it back to `quickstart.enabled=true` along with updates to pinot configs if any. 
Thanks for pointing this out!

Reply:
Added the timeout with descriptive log.